modules/ob-release-notes-1-2.adoc

```diff
@@ -0,0 +1,42 @@
+// This module is included in the following assemblies:
+// * about/ob-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ob-release-notes-1-2_{context}"]
+= Release notes for {builds-shortname} General Availability 1.1
+
+{builds-shortname} General Availability (GA) 1.1 is now available on {ocp-product-title} 4.16.
+
+[id="new-features-1-1_{context}"]
+== New features
+
+The following sections highlight what is new in {builds-shortname} 1.1.
+
+[id="builds-new-features-1-1_{context}"]
+=== Builds
+
+* The builds controllers now use Tekton's V1 API to create and access the `TaskRun` that backs a `BuildRun`.
+
+* With this release, you can now define a build without any source. This is useful if you want to run the build using only the local source.
+
+* The output image section now supports an optional timestamp field to change the image creation timestamp. Use the `SourceTimestamp` string to set it to match the source timestamp.
+
+* With this release, the `.source.type` field is now required in both `build` and `buildRun`.
+
+* With this release, the Operator now installs Shipwright by default and introduces several user experience enhancements for Builds.
```

Comment:
You cn delete the actual RNs and just keep this as a template for when the actual RNs will come :) 
```suggestion

```